jest/jest.setup.ts

```diff
@@ -58,3 +61,14 @@ jest.mock('libs/environment', () => ({
     ID_CHECK_URL: 'https://id-check-unit-tests',
   },
 }))
+
+export const originalConsoleError = console.error.bind(console.error)
+
+export function mockConsoleError(messageToIgnore: string) {
```

Comment:
Suggestion de naming :
```suggestion
export function ignoreConsoleError(messageToIgnore: string) {
```